navigation.yaml

```diff
@@ -447,6 +447,8 @@ legal:
           path: /legal/data-privacy/snap-store
         - title: Snapcraft NPS
           path: /legal/data-privacy/snapcraft-nps
+        - title: Partner portal
```

Comment:
"Partner Portal" or "Partner portal"?

Reply:
good catch, I assume "Partner Portal" is correct as the lawyers picked that